pkg/component/monitoring/bootstrap.go

```diff
@@ -0,0 +1,276 @@
+
+		applierOptions          = kubernetes.CopyApplierOptions(kubernetes.DefaultMergeFuncs)
+		retainStatusInformation = func(new, old *unstructured.Unstructured) {
+			// Apply status from old Object to retain status information
+			new.Object["status"] = old.Object["status"]
+		}
+	)
+
+	applierOptions[vpaGK] = retainStatusInformation
+	applierOptions[hvpaGK] = retainStatusInformation
+	applierOptions[issuerGK] = retainStatusInformation
+
+	var ingressTLSSecretName string
+	if b.values.WildcardCertName != nil {
+		ingressTLSSecretName = *b.values.WildcardCertName
+	} else {
+		ingressTLSSecret, err := b.secretsManager.Generate(ctx, &secretsutils.CertificateSecretConfig{
+			Name:                        "aggregate-prometheus-tls",
+			CommonName:                  "prometheus",
+			Organization:                []string{"gardener.cloud:monitoring:ingress"},
+			DNSNames:                    []string{b.values.IngressHost},
+			CertType:                    secretsutils.ServerCert,
+			Validity:                    pointer.Duration(v1beta1constants.IngressTLSCertificateValidity),
+			SkipPublishingCACertificate: true,
+		}, secretsmanager.SignedByCA(v1beta1constants.SecretNameCASeed))
+		if err != nil {
+			return err
+		}
+		ingressTLSSecretName = ingressTLSSecret.Name
+	}
+
+	values := kubernetes.Values(map[string]interface{}{
+		"global": map[string]interface{}{
+			"ingressClass": v1beta1constants.SeedNginxIngressClass,
+			"images": map[string]string{
+				"alertmanager":       b.values.ImageAlertmanager,
+				"alpine":             b.values.ImageAlpine,
+				"configmap-reloader": b.values.ImageConfigmapReloader,
+				"prometheus":         b.values.ImagePrometheus,
+			},
+		},
+		"prometheus": map[string]interface{}{
+			"resources":               monitoringResources["prometheus"],
+			"storage":                 b.values.StorageCapacityPrometheus,
+			"additionalScrapeConfigs": centralScrapeConfigs.String(),
+			"additionalCAdvisorScrapeConfigMetricRelabelConfigs": centralCAdvisorScrapeConfigMetricRelabelConfigs.String(),
+		},
+		"aggregatePrometheus": map[string]interface{}{
+			"resources":               monitoringResources["aggregate-prometheus"],
+			"storage":                 b.values.StorageCapacityAggregatePrometheus,
+			"seed":                    b.values.SeedName,
+			"hostName":                b.values.IngressHost,
+			"secretName":              ingressTLSSecretName,
+			"additionalScrapeConfigs": aggregateScrapeConfigs.String(),
+		},
+		"alertmanager": alertManagerConfig,
+		"hvpa": map[string]interface{}{
+			"enabled": b.values.HVPAEnabled,
+		},
+		"istio": map[string]interface{}{
+			"enabled": true,
+		},
+		"ingress": map[string]interface{}{
+			"authSecretName": b.values.GlobalMonitoringSecret.Name,
+		},
+	})
+
+	return b.chartApplier.ApplyFromEmbeddedFS(ctx, chart, chartPath, b.namespace, "monitoring", values, applierOptions)
+}
+
+func (b *bootstrapper) Destroy(ctx context.Context) error {
```

Comment:
Is the deletion of resources on seed deletion not required?

Reply:
Technically it is, but it wasn't implemented before, and this PR only focuses on moving the chart into the component package. It does not add new logic/behaviour. This cleanup topic should probably be followed-up separately.